client/store/animals.js

```diff
@@ -44,11 +48,22 @@ export const fetchAnimals = () => (dispatch) => {
     .catch(err => console.error('fetching animals unsucessful', err));
 };
 
-export const updateAnimal = (updateObj) => (dispatch) => {
+export const updateAnimal = updateObj => (dispatch) => {
   const id = updateObj.id;
   axios.put(`/api/admin/animals/${id}`, updateObj)
     .then((res) => {
       dispatch(update(res.data));
     })
     .catch(err => console.error('updating animal unsucessful', err));
-};
+
+}
+
+export const createAnimal = createObj => (dispatch) => {
+  axios.post('/api/admin/animals', createObj)
+    .then((res) => {
+      dispatch(create(res.data));
+      dispatch(fetchAnimals());
```

Comment:
you shouldn't have to fetch if create is just adding to the already fetched data